tests/integration/commands/invoke_command_test.py

```diff
@@ -36,8 +38,30 @@ async def test_invoke(
             wait_for_acceptance=True,
             gateway_url=devnet_gateway_url,
         )
+        await assert_transaction_accepted(devnet_gateway_url, response.transaction_hash)
 
+        response = await protostar.invoke(
```

Comment:
Could you please move this logic to separate test case?

Reply:
[address feedback in tests](https://github.com/software-mansion/protostar/pull/1140/commits/dd3248b2c6e8d02b1338c84e78c81b39e1ca0225)